mungegithub/mungers/blunderbuss.go

```diff
@@ -162,6 +162,19 @@ func selectMultipleOwners(potentialOwners weightMap, weightSum int64, count int)
 	return owners
 }
 
+func getPotentialOwnersHelper(author *string, feats *features.Features, files []*githubapi.CommitFile, issueNumber *int) (weightMap, int64) {
+	potentialOwners, weightSum := getPotentialOwners(*author, feats, files, true)
+	if len(potentialOwners) == 0 {
```

Comment:
I think it would be easier to read if you returned early. But that might just be my personal taste.

Reading the code further. I realize that you kept it as it was, which is not as bad. But now that it's in a separate function, it also makes it easier to return early.